libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -259,6 +261,7 @@ type SelfAPI =
   Named
     "get-self"
     ( Summary "Get your own profile"
+        :> Description "OAuth scope: `read:self`"
```

Comment:
I would have introduced a type `DescriptionOAuthScopes`.  Easier to refactor once we have something more correct-by-construction.